### PR TITLE
Rewrite content pages to use React

### PR DESCRIPTION
**Pull request by victorporof**
Fixes https://github.com/mozilla/tofino/issues/531 and https://github.com/mozilla/tofino/issues/493

### CODE REVIEW

**Comment by victorporof** [comment]
@bgrins

File: app/ui/content/index.jsx [hunk 1]
```diff
@@ -10,14 +10,20 @@
...
-}
+import App from './views/app';
+import { createContentStore } from './store';
 
-if (~document.location.pathname.indexOf('stars')) {
-  fetchRecentBookmarks();
-}
+const store = createContentStore();
+
+const chrome = (
```
Comment (bgrins) [inline app/ui/content/index.jsx]: 'chrome' name seems overloaded here, probably shouldn't show up at all in this app

Comment (victorporof) [inline app/ui/content/index.jsx]: Good point

File: app/ui/content/views/stars/star-item.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ */
+
+import React from 'react';
+
+import Style from '../../../shared/style';
+
+import * as ContentPropTypes from '../../model/content-prop-types';
+import ListItem from '../../../shared/widgets/list-item';
+
+const STAR_ITEM_STYLE = Style.registerStyle({
```
Comment (bgrins) [inline app/ui/content/views/stars/star-item.jsx]: This is duplicated between stars and history view.  I'd suggest move it to the shared ListItem component in LIST_ITEM_STYLE, until we have a third consumer that doesn't need this style at which point we could break it out.

**Comment by bgrins** [comment]
Looks good, thanks

**Pull request merged**